examples/arco-pro/package.json

```diff
@@ -38,9 +38,10 @@
   },
   "devDependencies": {
     "serve": "14.1.2",
-    "@rspack/plugin-react-refresh": "0.3.14",
-    "@rspack/cli": "0.3.14",
-    "@rspack/plugin-html": "0.3.14",
+    "@rspack/plugin-react-refresh": "0.4.0-canary-680a78e-20231120143513",
+    "@rspack/cli": "0.4.0-canary-680a78e-20231120143513",
+    "@rspack/core": "0.4.0-canary-680a78e-20231120143513",
```

Comment:
we need to install @rspack/core manually since it's no longer dependency of @rspack/cli